src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,104 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
 public class WorkWithFile {
+    static final int MAX_LINES = 100;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] rawFields = readToTable(fromFileName);
+        String[] sumFields = sumTogether(rawFields);
+        saveToFile(sumFields, toFileName);
+    }
+
+    public String [] readToTable(String fileName) {
+        String [] temporaryFields = new String[MAX_LINES];
```

Comment:
I don't think we need to limit the file size, let's use StringBuilder instead of array